mod/admin.php

```diff
@@ -782,6 +782,11 @@ function admin_page_workerqueue(App $a)
 	// get jobs from the workerqueue table
 	$statement = dba::select('workerqueue', ['id', 'parameter', 'created', 'priority'], ['done' => 0], ['order'=> ['priority']]);
 	$r = dba::inArray($statement);
+        
+	for($i = 0; $i < count($r); $i++) {
+		$r[$i]["parameter"] = implode(": ", explode("\",\"", $r[$i]["parameter"]));
```

Comment:
Code standards: Please use single quotes by default, and double quotes when you have to. For example this would read better as `'","'`.